client/my-sites/plans-features-main/index.jsx

```diff
@@ -274,7 +275,7 @@ export class PlansFeaturesMain extends Component {
 		return plans[ intervalType ] || defaultValue || TERM_ANNUALLY;
 	}
 
-	getPlansForPlanFeatures() {
+	getPlansForPlanFeatures( experimentVariation = null ) {
```

Comment:
Same as before, `null` is not needed as that's the default value.

Reply:
Thanks!